pkgs/development/libraries/blkhash/default.nix

```diff
@@ -0,0 +1,66 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchFromGitLab
+, makeWrapper
+, meson
+, ninja
+, pkg-config
+, cmake
+, openssl
+, enableLibnbd ? stdenv.isLinux, libnbd
+, qemu-utils
+, asciidoc
+, python3Packages
+}:
+let
+  unity = fetchFromGitHub {
+    owner = "ThrowTheSwitch";
+    repo = "Unity";
+    rev = "5204c1bacf37e0c38211d2fbb6d9796e410223f8";
+    sha256 = "i4uQOG/Q86AaQINJoC/65QA/ZEUOXzaVceR2inw32yU=";
+  };
+in
+stdenv.mkDerivation rec {
+  version = "0.6.1";
+  pname = "blkhash";
+
+  src = fetchFromGitLab {
+    owner = "nirs";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "+7SIZdvkdMYVgeklbqU0MHKc1VJnk9opqhsCmkg6bcI=";
+  };
+
+  postUnpack = ''(
+    cd "$sourceRoot/subprojects"
+    cp -R --no-preserve=mode,ownership ${unity} unity
+  )'';
+
+  dontUseCmakeConfigure = true;
+
+  buildInputs = [
+    makeWrapper
```

Comment:
The manual recommends to add makeWrapper in nativeBuildInputs.
https://nixos.org/manual/nixpkgs/stable/#fun-makeWrapper

Since the wrapper is created on the build machine we want the native version.